.github/workflows/pr-percy-prepare-label.yml

```
@@ -0,0 +1,22 @@
+# This workflow ensures Percy is executed against PRs with the Percy required label, regardless of which paths were changed
+name: "Percy [labeled] / Prepare build"
+
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+        # workflow runs after a label is added to the PR, or when a commit is added to a PR with the Percy label
+      - labeled
+      - ready_for_review
+      - synchronize
+
+jobs:
+  copy_artifact:
+    name: Copy changed files to GHA artifact
```

```suggestion
    name: Prepare build
```

These changes would need to be reflected in other places as well.

---

Revised